src/NServiceBus.Core/DelayedDelivery/DelayedDeliveryFeature.cs

```diff
@@ -12,7 +12,10 @@ public DelayedDeliveryFeature()
             DependsOnOptionally<TimeoutManager>();
             Defaults(s =>
             {
-                var timeoutManagerAddressConfiguration = new TimeoutManagerAddressConfiguration(s.GetConfigSection<UnicastBusConfig>()?.TimeoutManagerAddress);
+                var timeoutManagerAddressConfiguration =
+                    new TimeoutManagerAddressConfiguration(
+                        s.GetConfigSection<UnicastBusConfig>()?.TimeoutManagerAddress
```

Comment:
good point. I think it makes more sense to give the code first api precendence 👍 will change.